common-template/src/main/resources/templates/slack/Default/beta/default.json

```diff
@@ -0,0 +1,40 @@
+{
+    "blocks": [
+        {
+            "type": "header",
+            "text": {
+                "type": "plain_text",
+                "text": "{data.source.application.display_name} - {data.source.bundle.display_name}",
```

Comment:
I think we should also add the event type like `"*{data.source.event_type.display_name} - {data.source.application.display_name} - {data.source.bundle.display_name}*"`